app/assets/javascripts/member-facing/backbone/braintree_hosted_fields.js

```diff
@@ -45,6 +51,14 @@ const BraintreeHostedFields = Backbone.View.extend({
     };
   },
 
+  injectDeviceData(deviceData) {
+    let form = this.$('form')
```

Comment:
it's nice to prefix jquery variables with `$`. also we use semicolons per the [airbnb style guide](https://github.com/airbnb/javascript#semicolons)